SproxydKeysScan/Logging.js

```diff
@@ -0,0 +1,110 @@
+const werelogs = require('werelogs');
+const { env } = require('./env');
+
+/**
+ * @class
+ * @classdesc - aggregates events in a map and logs them at an interval.
+ * call .run() after instantiation to being loop.
+ */
+class AggregateLogger {
```

Comment:
I feel like these classes belong more in werelogs than in this repo.

Reply:
agree, but I thought it's better to validate the design and usage for this one specific task for some time before moving it into werelogs for general use.